`CSharpier.MsBuild` still targets .NETCoreApp,Version=v5.0 (somehow)

My guess is this line:
https://github.com/belav/csharpier/blob/e930e218a9b1b7e64f216ecdabc64be63ffd1ca5/Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj#L11

It causes an old version of csharpier cli that happens to be sitting around on the build machine to be bundled with the MsBuild nuget package. Causing this when someone tries to use it on a machine that only has `.net6` installed:
```
Microsoft (R) Build Engine version 17.0.0+c9eb9dd64 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
  Erabikata.Models -> /tmp/scratch/erab/backend/Erabikata.Models/bin/Debug/netstandard2.1/Erabikata.Models.dll
  Erabikata.Backend -> /tmp/scratch/erab/backend/Erabikata.Backend/bin/Debug/net6.0/Erabikata.Backend.dll
  It was not possible to find any compatible framework version
  The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
    - The following frameworks were found:
        6.0.0 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  
  You can resolve the problem by installing the specified framework and/or SDK.
  
  The specified framework can be found at:
    - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=arch-x64
/home/fernie/.nuget/packages/csharpier.msbuild/0.14.0/build/CSharpier.MsBuild.targets(11,9): error MSB3073: The command "dotnet /home/fernie/.nuget/packages/csharpier.msbuild/0.14.0/build/../tools/csharpier/dotnet-csharpier.dll  /tmp/scratch/erab/backend/Erabikata.Backend" exited with code 150. [/tmp/scratch/erab/backend/Erabikata.Backend/Erabikata.Backend.csproj]

Build FAILED.

/home/fernie/.nuget/packages/csharpier.msbuild/0.14.0/build/CSharpier.MsBuild.targets(11,9): error MSB3073: The command "dotnet /home/fernie/.nuget/packages/csharpier.msbuild/0.14.0/build/../tools/csharpier/dotnet-csharpier.dll  /tmp/scratch/erab/backend/Erabikata.Backend" exited with code 150. [/tmp/scratch/erab/backend/Erabikata.Backend/Erabikata.Backend.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:03.00
```

--- a/Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj
+++ b/Src/CSharpier.MsBuild/CSharpier.MsBuild.csproj
@@ -8,7 +8,7 @@
         -->
         <TargetFramework>netstandard2.0</TargetFramework>
         <PackageId>CSharpier.MsBuild</PackageId>
-        <CSharpierOutputDir>../CSharpier.Cli/bin/$(Configuration)/net5.0</CSharpierOutputDir>
+        <CSharpierOutputDir>../CSharpier.Cli/bin/$(Configuration)/net6.0</CSharpierOutputDir>
     </PropertyGroup>
 
     <!--

